pages/simple.vue

```diff
@@ -1,131 +1,136 @@
+        <input class="winter-slider__slider" ref="slider" type="range" min="-0.3" max="0.2" step="0.01" v-model.number="scenarioFactor">
+      </div>
+      <div class="app-content-block __result">
+        <p>Je geschatte kosten in 2023 met het prijsplafond zijn voor gas <strong>&euro;&nbsp;{{ overzicht.gasJaarKosten }}</strong> en
+          voor elektriciteit <strong>&euro;&nbsp;{{ overzicht.kWhJaarKosten }}</strong>. Per maand is dat voor gas <strong>&euro;&nbsp;{{
+              overzicht.gasMaandKosten
+            }}</strong> en voor elektriciteit <strong>&euro;&nbsp;{{ overzicht.kWhMaandKosten }}</strong>.
         </p>
-      </section>
-
-      <section class="app-content-block section-verantwoording">
-        <h2 class="app-section__heading">Verantwoording en achtergrond</h2>
         <ul>
-          <li>
-            <NuxtLink to="/verantwoording">Verantwoording</NuxtLink>
-          </li>
+          <li><strong>Kosten per jaar: &euro;&nbsp;{{ overzicht.jaarKosten }}</strong></li>
+          <li><strong>Kosten per maand: &euro;&nbsp;{{ overzicht.maandKosten }}</strong></li>
         </ul>
-      </section>
-    </div>
+      </div>
+
+      <div class="app-content-block __result">
+        <p>Een zeer koude winter verhoogt het gasverbruik met 30%. De kans op een zeer koude winter is klein, maar de gevolgen kunnen groot zijn. Een milde winter scheelt verwarmingskosten. De invloed daarvan is kleiner: het scheelt maximaal 20% van de verwarmingskosten.</p>
+      </div>
+    </section>
   </main>
 </template>
+
+<script setup>
   
-  <script setup>
-  
-  import { useSimpleComputationStore } from '~~/simple-store'
-  import { storeToRefs } from 'pinia';
-  
-    const store = useSimpleComputationStore()
-    const { 
-        gasPlafond, gasPlafondTarief, gasJaarVerbruik, gasJaarTarief,
-        kWhPlafond, kWhPlafondTarief, kWhJaarVerbruik, kWhJaarTarief,
-        kWhJaarTeruglevering, kWhJaarTerugleververgoeding, kWhJaarTerugLeveringToggle,
-        voorschotPerMaand, 
-        scenarioFactor, scenarioEffectOpStroom, 
-        overzicht, validate
-    } = storeToRefs(store)
-  
-  </script>
+import { useSimpleComputationStore } from '~~/simple-store'
+import { storeToRefs } from 'pinia';
+import { ref, watch } from "vue";
+
+const store = useSimpleComputationStore()
+const {
+    gasPlafond, gasPlafondTarief, gasJaarVerbruik, gasJaarTarief,
+    kWhPlafond, kWhPlafondTarief, kWhJaarVerbruik, kWhJaarTarief,
+    kWhJaarTeruglevering, kWhJaarTerugleververgoeding, kWhJaarTerugLeveringToggle,
+    voorschotPerMaand,
+    scenarioFactor, scenarioEffectOpStroom,
+    overzicht, validate
+} = storeToRefs(store)
+
+watch(kWhJaarTerugLeveringToggle, (value) => {
+  if (value) {
+    kWhJaarVerbruik.value = 0
+  }
+})
+
+watch(kWhJaarVerbruik, (value) => {
+  if (value < 0) {
+    kWhJaarTerugLeveringToggle.value = true
+    kWhJaarTeruglevering.value = value * -1
+
+    // TODO: focus op de terug leverings input box zetten
```

Comment:
Dit is nog een todo, daarna is het denk ik klaar. idee is dat als je negatieve stroom invult dat de focus naar de juiste input box verandert.